Update instructions as per latest vivaldi version

jfchevrette opened the pull request:
A few changes so that the instructions match the latest vivaldi version settings UI. For instance, the menu as instructed at the first step is completely gone.

This also reword a few things and adds a note for people who want the UI colors to be consistent with Dracula colors instead of letting Vivaldi colorize the tabs depending on the websites colors

jfchevrette committed:
update instructions as per latest vivaldi version

screendriver commented:
> For instance, the menu as instructed at the first step is completely gone

No. On macOS everything is still there as described. What you are describing in your pull request is for the Windows version 😉

zenorocha commented:
@jfchevrette can you change your pull request to include two sections? One for Mac and one for Windows?

jfchevrette committed:
update instructions

jfchevrette commented:
@screendriver you are totally right. Apologies for that. I've added separate instructions for Windows, macOS and Linux after I tested on all 3 platforms

@zenorocha Done. I've split item 1 into 3 bullets instead of creating 3 sections with only the first step being different. From looking at other app's instructions, I think the bullets will show nicely on the website. Let me know if you prefer otherwise.

jfchevrette committed:
small cleanup

screendriver reviewed (approved):
LGTM 👍
